src/main/webapp/pages/remote_apis/parent_reload.html

```diff
@@ -1,15 +1,24 @@
 <!DOCTYPE html>
 <html>
-<body>
-
-    <script>
+  <body>
+    <script th:inline="javascript">
       /*
-      This is loaded into an iFrame.
-      Close the bootstrap modal on the parent page.
-       */
-      var modal = window.parent.document.querySelector("#remote-connect-wrapper");
-      modal.dataset.result = "success";
-      modal.querySelector(".close").click();
+  This is loaded into an iFrame.
+  Close the bootstrap modal on the parent page.
+   */
+      const modal = window.parent.document.querySelector(
+        "#remote-connect-wrapper"
+      );
+      if (modal) {
```

Comment:
Add a comment here of what it's doing and why

Reply:
Add in 99f9d8160b81df0946290cba745f43c42119ba8e